mp/src/game/shared/momentum/weapon/weapon_mom_shotgun.cpp

```diff
@@ -83,13 +83,13 @@ void CMomentumShotgun::PrimaryAttack()
 #endif
         m_flPumpTime = gpGlobals->curtime + 0.5;
 
-    m_flNextPrimaryAttack = gpGlobals->curtime + 0.25;
-    m_flNextSecondaryAttack = gpGlobals->curtime + 0.25;
+    m_flNextPrimaryAttack = gpGlobals->curtime + PrimaryFireTime();
+    m_flNextSecondaryAttack = gpGlobals->curtime + PrimaryFireTime();
 #ifdef WEAPONS_USE_AMMO
     if (m_iClip1 != 0)
         SetWeaponIdleTime(gpGlobals->curtime + 2.5);
     else
-        SetWeaponIdleTime(gpGlobals->curtime + 0.25);
+        SetWeaponIdleTime(gpGlobals->curtime + PrimaryFireTime());
 #else
     SetWeaponIdleTime(gpGlobals->curtime + 2.5f);
```

Comment:
You may want to make this 2.5f a const for the function, or a `define` up top if the value (and its context) is used outside of this function, I'm seeing it in multiple places.

Reply:
Consider it done